### ISSUE TITLE
Resource Request

### ISSUE THREAD

**Issue by athulmanuel** [coronasafe/care_fe#9419]
add category of ' Financial' to the list of categories in Resource Request

@vigneshhari

**Comment by bodhish** [comment]
@sainak do verify if this is done, I think its done. 
Also I have seen a bug when other is selected on the FE, just confirm if its fixed.

**Comment by sainak** [comment]
yes bodhi this was added in https://github.com/ohcnetwork/care/commit/10aaf748b5afc8e9cbe5e5cd43e177186d4eac5f

**Comment by Jacobjeevan** [comment]
@sainak Resource [model](ttps://github.com/ohcnetwork/care/commit/10aaf748b5afc8e9cbe5e5cd43e177186d4eac5f#diff-ee6c11d8644dcf087b1f2b2caf09148b05a8530c2adc581210c09f021dd9b86c) is adding `others` as the category, while FE is sending `other`.

 Of course, could just adjust that on FE side, not sure how important that is - if it's coming from FHIR spec; I'm just seeing codeable concept for [ServiceRequest](https://www.hl7.org/fhir/servicerequest.html).

**Comment by bodhish** [comment]
@Jacobjeevan This is our custom implementation, I would recommend that we update FE to match the BE and close this issue.

**Comment by Jacobjeevan** [comment]
> [@Jacobjeevan](https://github.com/Jacobjeevan) This is our custom implementation, I would recommend that we update FE to match the BE and close this issue.

Yea ended up doing that; displaying "Others" on FE made more sense (but passing other to BE).
